src/components/AgenciesPage/services/getAgenciesData.ts

```diff
@@ -5,16 +5,21 @@ import {
   fetchAgenciesSuccess,
   fetchAgenciesFailure,
 } from '../actions/actionCreators';
+import { errorAlertSetErrorMessage } from '../../../shared/components/ErrorAlert/actions/errorAlertActions';
+import type { ErrorAlertAllActions } from '../../../shared/components/ErrorAlert/types/errorAlertTypes';
 import type { AgenciesAction, Agency } from '../types/Agencies';
 
 export const getAgenciesData =
   () =>
-  async (dispatch: Dispatch<AgenciesAction>): Promise<void> => {
+  async (
+    dispatch: Dispatch<AgenciesAction | ErrorAlertAllActions>,
+  ): Promise<void> => {
     dispatch(fetchAgenciesRequest());
     try {
       const response: Agency[] = await fetchAgencies();
       dispatch(fetchAgenciesSuccess(response));
     } catch (error) {
       dispatch(fetchAgenciesFailure((error as Error).message));
```

Comment:
Here, and everywhere you don't need to pass an error message already in fetchAgenciesFailure, etc, because you pass it in alert

Reply:
fixed